repair_type/tests/test_repair_type.py

```diff
@@ -6,12 +6,13 @@
 
 class TestRepairType(TransactionCase):
     def setUp(self, *args, **kwargs):
```

Comment:
Please use `setUpClass`

Reply:
`setUp`  is used here because we need to  reset the common values before to run every test. The tests are checking if the source and destination locations are correctly set after setting `repair_type_id`. So we need to have this value unsetted at the start of every test. 

`setUpClass` is used when you need to only run once for all tests and `setUp` is executed before every test.